policyengine_us/parameters/gov/states/dc/doee/liheap/income_level_increment.yaml

```
@@ -0,0 +1,12 @@
+description: District of Columbia determines Low Income Home Energy Assistance Program payout for Heat in Rent Utility Type
```

Active voice please.
```suggestion
description: District of Columbia determines Low Income Home Energy Assistance Program payout amounts in these increments of [income concept] for ....
```

Not sure what the $2,000 increment stands for